client/reader/stream/index.jsx

```diff
@@ -500,7 +501,12 @@ class ReaderStream extends Component {
 					renderLoadingPlaceholders={ this.renderLoadingPlaceholders }
 				/>
 			);
-			const sidebarContent = <ReaderListFollowedSites path={ path } />;
+
+			const sidebarContent = path.startsWith( '/tag/' ) ? (
```

Comment:
nit: there would probably be a neater way to check this like passing it in as a param